js/count_monkeys.js

```diff
@@ -0,0 +1,13 @@
+// https://www.codewars.com/kata/count-the-monkeys/train/javascript
+
+function monkeyCount(n) {
+  const array = [];
```

Comment:
Nice, it seems to work 👌 

1. Could we use a `for` loop instead of `do ... while`? It could be more readable here
2. Could we use [Array.from](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from) with the `mapFn` supplied?

A neat trick using `Array.from` when constructing an array containing N items is:

```ts
Array.from({ length: N })
```

try using it in point 2

Reply:
I've tried both options.